packages/client/tests/functional/issues/9007/tests.ts

```diff
@@ -8,19 +9,23 @@ declare let prisma: PrismaClient
 
 // https://github.com/prisma/prisma/issues/9007
 testMatrix.setupTestSuite(
-  () => {
-    test('should throw an error if using contains filter on uuid type', async () => {
-      await prisma.user.create({ data: {} })
+  ({ providerFlavor }) => {
+    // TODO it seems like adapters cannot handle uuid. Unsupported column type: 2950
```

Comment:
```suggestion
    // TODO it seems like adapters cannot handle uuid. Unsupported column type: 2950
    // tracked in https://github.com/prisma/team-orm/issues/374
```

Reply:
Fixed in https://github.com/prisma/prisma-engines/pull/4290.